### DISCUSSION TITLE
Codepoint symbols?

### DISCUSSION THREAD

**Discussion by alkaris2** [kovidgoyal/kitty#3553] [Q&A]
Kitty complains that symbol map for codepoint <font_name> is invalid, ignoring.

It doesn't seem to matter what font I specify it always complains its invalid. If I try to use `PowerlineSymbols` codepoint it says its invalid and ignores it.

![image](https://user-images.githubusercontent.com/73155407/116433004-41546280-a888-11eb-80b7-f33f5d20e6f7.png)

I don't get how this is suppose to work, and there's little to no documentation on this of how you're supposed to define this, that one comment section there in the file doesn't explain enough, or give a clear example of how you're supposed to define this configuration. Don't you have to specify the font name and then the symbols you want to map below it? or just specify the font and it takes the symbol mappings from that font automatically?

**Comment by Luflosi** [comment]
For example:
```
symbol_map U+23FB-U+23FE,U+2B58,U+E200-U+E2A9,U+E0A0-U+E0A3,U+E0B0-U+E0BF,U+E0C0-U+E0C8,U+E0CC-U+E0CF,U+E0D0-U+E0D2,U+E0D4,U+E700-U+E7C5,U+F000-U+F2E0,U+2665,U+26A1,U+F400-U+F4A8,U+F67C,U+E000-U+E00A,U+F300-U+F313,U+E5FA-U+E62B Symbols Nerd Font
```

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf.html#opt-kitty.symbol_map
